tests/test_coiled.py

```diff
@@ -0,0 +1,37 @@
+import sys
+import uuid
+
+import coiled
+import dask.dataframe as dd
+import pandas as pd
+from dask.distributed import Client
+
+SOFTWARE = f"dask-engineering/coiled_dist-py{sys.version_info[0]}{sys.version_info[1]}"
+
+
+def test_quickstart():
+    with coiled.Cluster(
```

Comment:
@ncclementi I'd encourage you to use the new ("declarative") clusters:

`from coiled.v2 import cluster`

If you let me know what Coiled account you're using, I'll make sure it has permission to use these.